packages/components/src/breadcrumbs/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+const dashboardSharedConfig = require( '../../.eslintrc.dashboard-shared.js' );
```

Comment:
It felt odd that a package would require this kind of configuration for the code outside the package...

Reply:
Yes, but didn't we decide that this was an ok compromise that would make us more confident about importing components from `@automattic/components`?

Reply:
Yeah, but I think we can rephrase the sentence to emphasize that the goal is to keep the dependencies of these components simpler and cleaner, and that we'll gradually apply similar changes to other components or move them to the ui package where appropriate 🙂

What do you think?

Reply:
I'm just going through my stale PRs, and thought about tidying up this one.

> I think we can rephrase the sentence to emphasize that the goal is to keep the dependencies of these components simpler and cleaner

I don't think I understand what you mean here. Are you saying we could have a better warning message?

Reply:
I think it's OK to refer to dashboard, otherwise the reader will be confused what's the context about this comment. It's the tradeoff (otherwise we shouldn't be doing this). Anyway this package will be replaced by `@automattic/ui`. 😅